java/src/main/java/com/google/oak/stub/QueueingStreamObserver.java

```diff
@@ -0,0 +1,79 @@
+//
```

Comment:
I think this file could also be moved to `transport` instead of creating a separate `stub` directory only for this file

Reply:
My understanding is that transparent contains classes that are meant to be used in the client. But `stub` is more generic than that and is used in server too. I also decided to reuse the name `stub` from `io.grpc`, because this class is an extension of `io.grpc.stub.StreamObserver`.

But if `transport` is meant to be more generic than client-side utilities, I can move this class there. Let me know.

Reply:
I think `transport` can contain more generic transportation related code, since it's not inside the `java/.../oak/client` directory.

Reply:
Merged `stud` into `transport`.